modules/react-mapbox/src/types/common.ts

```diff
@@ -27,6 +27,8 @@ export type ViewState = {
   pitch: number;
   /** Dimensions in pixels applied on each side of the viewport for shifting the vanishing point. */
   padding: PaddingOptions;
+  /** Center elevation on terrain */
+  altitude?: number;
```

Comment:
Could we call this elevation since it represents the ground level? Altitude can be reserved for camera position up in the air.